rmgpy/kinetics/arrhenius.pyx

```diff
@@ -249,7 +249,23 @@ cdef class Arrhenius(KineticsModel):
 
         # Set the rate parameter to a cantera Arrhenius object
         ctReaction.rate = self.toCanteraKinetics()
-    
+
+    cpdef ArrheniusEP toArrheniusEP(self):
```

Comment:
Yes you could do it, and it may well have good uses, but you would need to pass it ∆H as well as alpha if alpha ≠ 0. But would you enforce ∆H is always passed to this function, or let it be omitted if alpha is also omitted?